curriculum/challenges/english/07-scientific-computing-with-python/learn-string-manipulation-by-building-a-cipher/6553efd6ada3f42aa2d75448.md

```diff
@@ -16,10 +16,20 @@ def function_name(param_1, param_2):
 
 Parameters are variables that you can use inside your function. A function can be declared with different number of parameters. In the example above, `param_1` and `param_2` are parameters.
 
-Modify your function declaration so that it takes two parameters called `message` and `offset`.
+First, comment out the function call to avoid an error.
+
```

Comment:
We should not ask to comment out the function call. I would go with something like:

"Modify your function declaration so that it takes two parameters called `message` and `offset`. After that, you'll see an error appear in the terminal."

Reply:
I agree, the cause of the error should be explained here instead of step 53.
In step 53, it can be mentioned that "`Now you can see the error has disappeared as the required number of arguments have been passed`".